src/common/perf_counters.h

```diff
@@ -233,6 +233,7 @@ class PerfCounters
   void dec(int idx, uint64_t v = 1);
   void set(int idx, uint64_t v);
   uint64_t get(int idx) const;
+  std::pair<uint64_t, uint64_t> get_avg(int idx) const;
```

Comment:
strictly speaking, this change does not belong to the fix, i'd suggest move it into its own commit.

Reply:
i knew. but the reason you added it is actually for testing the fix for `get_tavg_ns()`. ideally, we should be testing `get_tavg_ns()` instead, no?

if there is no consumers of an API, we intend to avoid adding it.

Reply:
Sorry, I did't notice that get_tavg_ns() exists already. I worked on jewel before.